### PR TITLE
Added spm tracing support

### PR DESCRIPTION
**Pull request by qkoshelev**
Added tracing support for spm-kamon and communication with sematext server in thrift format

### CODE REVIEW

**Comment by dpsoft** [comment]
@qkoshelev   thanks for proposing this addition to Kamon :D, please sign our [CLA] and we are good to go!

[CLA]: https://docs.google.com/forms/d/1G_IDrBTFzOMwHvhxfKRBwNtpRelSa_MZ6jecH8lpTlc/viewform

**Comment by qkoshelev** [comment]
Sorry for delay. Done.

2016-07-08 18:20 GMT+04:00 Diego Parra <notifications@github.com>:

File: kamon-spm/src/main/resources/reference.conf [hunk 1]
```diff
@@ -16,6 +17,7 @@ kamon {
       akka-dispatcher = [ "**" ]
       akka-router     = [ "**" ]
       system-metric   = [ "**" ]
+      # trace           = [ "**" ]
```
Comment (dpsoft) [inline kamon-spm/src/main/resources/reference.conf]: this is ok?

Comment (otisg) [inline kamon-spm/src/main/resources/reference.conf]: I think this was left commented out by mistake, right @qkoshelev ?

**Pull request merged**